website/docs/r/storage_sync_cloud_endpoint.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+subcategory: "Storage"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_sync_cloud_endpoint"
+description: |-
+  Manages a Storage Sync Cloud Endpoint.
+---
+
+# azurerm_storage_sync_cloud_endpoint
+
+Manages a Storage Sync Cloud Endpoint.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_sync" "example" {
+  name                = "example-ss"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_storage_sync_group" "example" {
+  name            = "example-ss-group"
+  storage_sync_id = azurerm_storage_sync.example.id
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "example-stracc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_storage_share" "example" {
+  name                 = "example-share"
+  storage_account_name = azurerm_storage_account.example.name
+  lifecycle {
+    ignore_changes = [
+      acl, metadata
```

Comment:
we shouldn't be shipping this in examples - what extra configuration is necessary here?

Reply:
There'll be a storage sync signature assigned to the metadata, which we could not get from the storage sync service. Could we just ignore the metadata?

Reply:
And as for `acl`, it's assigned to the following when a cloud endpoint is created, However, in share definition, `start` and `expiry` is necessary and no empty string requested:
``` 
 acl {
    id = "GhostedRecall"
    access_policy {
      permissions = "r"
    }
  }
```

Reply:
I agree with tom, if we are ignoring changes we need to fix the resources so its not required.

Reply:
Sure. Create a PR for the fix https://github.com/terraform-providers/terraform-provider-azurerm/pull/8811